### PR TITLE
fix: response type on resolve_market_reasoning.py

### PR DESCRIPTION
**Pull request by 0xArdi**
Hot fix to change the response type of `resolve_market_reasoning`.

### CODE REVIEW

File: packages/napthaai/customs/resolve_market_reasoning/resolve_market_reasoning.py [hunk 1]
```diff
@@ -700,7 +700,7 @@ def run(**kwargs) -> Tuple[Optional[str], Optional[Dict[str, Any]], Any]:
         print(f"Valid: {valid_results}")
 
         if not valid_results.is_valid:
-            return valid_results.json(), None, None, None, None
+            return valid_results.json(), None, None, None
```
Comment (0xArdi) [inline packages/napthaai/customs/resolve_market_reasoning/resolve_market_reasoning.py:703]: interface MUST be respected

Comment (jmoreira-valory) [reply packages/napthaai/customs/resolve_market_reasoning/resolve_market_reasoning.py:703]: I see only three arguments in the interface:

`def run(**kwargs) -> Tuple[Optional[str], Optional[Dict[str, Any]], Any]:
`
what I am missing?

**Review by angrybayblade — APPROVED** [review]
LGTM, except the formatting is a bit weird

File: packages/napthaai/customs/resolve_market_reasoning/resolve_market_reasoning.py (hunk 1 above)
Comment (jmoreira-valory) [reply packages/napthaai/customs/resolve_market_reasoning/resolve_market_reasoning.py:705]: @0xArdi According to current README.md:

> Such file must contain a `run` function that accepts `kwargs` and must always return a tuple `(Tuple[Optional[str], Optional[Dict[str, Any]], Any, Any])`.

File: packages/napthaai/customs/resolve_market_reasoning/resolve_market_reasoning.py [hunk 2]
```diff
@@ -35,10 +35,17 @@
...
 from googleapiclient.discovery import build
+from tiktoken import encoding_for_model
 
 client: Optional[OpenAI] = None
 
 
+def count_tokens(text: str, model: str) -> int:
+    """Count the number of tokens in a text."""
+    enc = encoding_for_model(model)
+    return len(enc.encode(text))
```
Comment (0xArdi) [inline packages/napthaai/customs/resolve_market_reasoning/resolve_market_reasoning.py:46]: was missing

File: packages/napthaai/customs/resolve_market_reasoning/resolve_market_reasoning.py [hunk 3]
```diff
@@ -700,121 +711,117 @@ def run(**kwargs) -> Tuple[Optional[str], Optional[Dict[str, Any]], Any]:
...
+        print(f"Results: {results}")
+
+        if counter_callback is not None:
+            counter_callback(
+                input_tokens=response_reasoning.usage.prompt_tokens
+                             + response_prediction.usage.prompt_tokens,
+                output_tokens=response_reasoning.usage.completion_tokens
+                              + response_prediction.usage.completion_tokens,
+                model=engine,
+                token_counter=count_tokens,
```
Comment (0xArdi) [inline packages/napthaai/customs/resolve_market_reasoning/resolve_market_reasoning.py:819]: was missing

**Review by angrybayblade — APPROVED** [review]

**Pull request merged**